stack/lambdas/iiif/lib/authorize.js

```diff
@@ -75,19 +83,32 @@ async function authorize(token, id, referer) {
   }
 
   var currentUser = getCurrentUser(token);
-  var docUrl = url.parse(url.resolve(elasticSearch, `common/_doc/${id}`));
-  var request = await makeRequest('GET', docUrl);
+  var query = {
+    query: {
+      match: { _id: id },
+    },
+  };
+
+  var docUrl = url.parse(url.resolve(elasticSearch, `_all/_search`));
+  var request = await makeRequest("POST", docUrl, JSON.stringify(query));
   var response = await fetchJson(request);
   var doc = response.json;
 
-  if (isObject(doc._source) && isString(doc._source.visibility)) {
-    switch(doc._source.visibility) {
-      case 'open':          return true;
-      case 'authenticated': return isObject(currentUser);
-      case 'restricted':    return false;
+  if (
+    isObject(doc.hits) &&
+    doc.hits.total >= 1 &&
+    isString(doc.hits.hits[0]._source.visibility)
+  ) {
+    switch (doc.hits.hits[0]._source.visibility) {
```

Comment:
In light of the recent `open`/`OPEN` thing, should we put a `toLowerCase()` on this switch? Or just trust the indexer?